tests/unit/config/test_normalize.py

```diff
@@ -69,27 +69,27 @@
         ),
         (
             {'location': {'repositories': ['foo@bar:/repo']}},
-            {'location': {'repositories': ['ssh://foo@bar/repo']}},
+            {'location': {'repositories': [{'path': 'ssh://foo@bar/repo', 'label': ''}]}},
```

Comment:
Similar thing here. (It probably would require changing what this function outputs though.)